src/lib/y2ftp/clients/ftp_server.rb

```diff
@@ -19,7 +19,6 @@
 
 require "yast"
 
-Yast.import "FtpServer"
```

Comment:
this is clearly wrong. If import has side-effect then please add it to initialize method with note that import has side-effects.

Reply:
This `Yast.import "FtpServer"` was not here initially, but I added it because IMHO this client should import it. But importing it here makes the testing very difficult, because there is no way to mock the importation of the FtpServer module nor the modules imported by FtpServer (as Users). For this reason I have removed it again.

But I agree, maybe we could simply add it to the initializer.

Reply:
why it is not possible to mock it. Can you explain it? Usually what is done in project is having something like
```
stub_module("A")
stub_module("B")
stub_module("C")
```
in test_helper and in specific test you have 
```
require_relative "../test_helper"

require <testing code>
```
so it is already mocked.
```

Reply:
Well, yes, you can stub the object before requiring the file, but I don't like it too much. I prefer to move the `Yast.import` calls inside the constructor.